### DISCUSSION TITLE
Found a new Sphinx extension for redirects

### DISCUSSION THREAD

**Discussion by choldgraf** [executablebooks/meta#117] [Ideas]
Just came across this (new) extension and it seems useful! Potentially something we could recommend in JB? https://github.com/wpilibsuite/sphinxext-rediraffe

**Comment by jstac** [comment]
Awesome project logo `:-)`

**Comment by mmcky** [comment]
Oh this would have been useful in our project migrations at `quantecon` :-)

**Reply by chrisjsewell** [reply]
It will be, now that I've fixed it up 😉 https://github.com/wpilibsuite/sphinxext-rediraffe/pull/17, https://github.com/wpilibsuite/sphinxext-rediraffe/pull/19
